### ISSUE TITLE
Game Crashing While Joining A World Or Server.

### ISSUE THREAD

**Issue by RohanDaCoder** [cabaletta/baritone#4272]
## Some information
Operating system: Android (Pojav Launcher)
Java version: 1.8.0_392
Minecraft version: 1.15.2 (Forge + Optifine)
Baritone version: baritone-standalone-1.5.3
Other mods (if used): None
I will upload my latestlog.txt, and the crash report.
[latestlog.txt](https://github.com/cabaletta/baritone/files/14232851/latestlog.txt)
[crash-2024-02-11_21.06.42-client.txt](https://github.com/cabaletta/baritone/files/14232852/crash-2024-02-11_21.06.42-client.txt)
So Basically,
When i Join A World/Server While Using Baritone, My Game Crashes.. I have tested it without baritone and it works fine, no issues.
I am using Pojav Launcher To Run Minecraft..
Some extra info: (probably useless)
Device: Vivo Y11
CPU: Snapdragon 439 (64 Bit)
Discord: rohan_ohio
Please contact me if anything is needed to fix this issue..
Thanks.

**Comment by ZacSharp** [comment]
Are you sure you are using the forge version of Baritone?
